### PR TITLE
Asset module migration

### LINKED ISSUE #232: The `AssetDetails` type updated in `v0.9.36`
https://github.dev/paritytech/substrate/blob/cb4f2491b00af7d7817f3a54209c26b20faa1f51/frame/assets/src/migration.rs#L27

The state processor rely on this struct, adoption needs here.

### PR DESCRIPTION
**Pull request by boundless-forest**
Close #232 

I am still not sure how to deal with the balance migration, please review other changes first.

### CODE REVIEW

**Comment by AurevoirXavier** [comment]
Only Pangolin need these I think.

**Comment by boundless-forest** [comment]
> Only Pangolin need these I think.

I think you're right. Let me fix it later.

File: runtime/crab/src/lib.rs [hunk 1]
```diff
@@ -93,8 +93,14 @@ pub type Executive = frame_executive::Executive<
 	frame_system::ChainContext<Runtime>,
 	Runtime,
 	AllPalletsWithSystem,
+	Migrations,
 >;
 
+type Migrations = (
+	// pallet_balances::migration::MigrateToTrackInactive<Runtime, CheckingAccount>,
```
Comment (boundless-forest) [inline runtime/crab/src/lib.rs:100]: From https://github.com/paritytech/substrate/pull/12813
> For chains with XCM, this should be used to mark the Checking Account when upgraded.

@jiguantong @AurevoirXavier Please help me. I am not sure if we should apply the `MigrateToTrackInactive` migration in this case.

**Comment by boundless-forest** [comment]
@AurevoirXavier Please review again. The commit history is a bit messy. :smile:

File: tool/state-processor/src/type_registry.rs [hunk 2]
```diff
@@ -62,7 +62,21 @@ pub struct AssetDetails {
...
+}
+
+// https://github.com/paritytech/substrate/blob/polkadot-v0.9.36/frame/assets/src/types.rs#L35
+#[derive(Debug, Encode, Decode, PartialEq)]
+pub enum AssetStatus {
+	Live,
+	Frozen,
+	Destroying,
+}
+
```
Comment (AurevoirXavier) [inline tool/state-processor/src/type_registry.rs:75]: Suggest removing this empty line.

**Review by AurevoirXavier — DISMISSED** [review]

**Pull request merged**